friktion/main.py

```diff
@@ -101,32 +101,32 @@ async def main_def():
     )
 
     assert swap_order_pre_fill.status == Created()
+    swap_order_creator = swap_order_pre_fill.creator
     order_id = swap_order_pre_fill.order_id
 
-    offer_pre_fill: Offer = await c.get_offer_details(wallet.public_key, order_id)
+    offer_pre_fill: Offer = await c.get_offer_details(swap_order_creator, order_id)
     print(f'order post fill: {offer_pre_fill}')
 
     print('2. taker executes bid against offer...')
 
     bid_details = BidDetails(
-        wallet.public_key,
-        order_id,
-        creator_give_pool_key,
-        creator_receive_pool_key,
-        1,
-        1,
+        order_id=order_id,
+        counterparty_give_pool=creator_give_pool_key,
```

Comment:
is that correct passing the creator pools as the counterparty pools? Shouldn't it be the buyer (signer) wallet?

For this testing script it does not make much difference as the wallets are the same, but it will affect the refactoring I am doing. I am moving this pool fetching to the validate_bid function

Reply:
good point! The creator and counterparty are identical in main.py, but that should definitely be the counterparty_give_pool